### ISSUE TITLE
deeplabv3p_xception65_optic.yaml训练问题！！

### ISSUE THREAD

**Issue by jackie8310** [PaddlePaddle/PaddleSeg#589]
利用deeplabv3p_xception65_optic.yaml，数据效验OK，训练时一直停留在这个样子，求解？？？
![image](https://user-images.githubusercontent.com/63913709/99471230-2b5d8000-2981-11eb-92a5-431d29e40048.png)

**Comment by wangkan83** [comment]
![image](https://user-images.githubusercontent.com/63913709/99471375-5fd13c00-2981-11eb-85ff-2101b1e1f24e.png)

**Comment by nepeplwu** [comment]
@wangkan83 你的训练命令麻烦给下

**Comment by jackie8310** [comment]
一直停留在这里，GPU显存仍在运行，不过用了一半的显存。
(paddle_env) G:\paddle\PaddleSeg\release-v0.7.0>python pdseg/train.py --cfg configs/deeplabv3p_xception65_xiwa.yaml --use_gpu --do_eval
{'AUG': {'AUG_METHOD': 'unpadding',
         'FIX_RESIZE_SIZE': (512, 512),
         'FLIP': False,
         'FLIP_RATIO': 0.5,
         'INF_RESIZE_VALUE': 500,
         'MAX_RESIZE_VALUE': 600,
         'MAX_SCALE_FACTOR': 2.0,
         'MIN_RESIZE_VALUE': 400,
         'MIN_SCALE_FACTOR': 0.5,
         'MIRROR': True,
         'RICH_CROP': {'ASPECT_RATIO': 0.33,
                       'BLUR': False,
                       'BLUR_RATIO': 0.1,
                       'BRIGHTNESS_JITTER_RATIO': 0.5,
                       'CONTRAST_JITTER_RATIO': 0.5,
                       'ENABLE': False,
                       'MAX_ROTATION': 15,
                       'MIN_AREA_RATIO': 0.5,
                       'SATURATION_JITTER_RATIO': 0.5},
         'SCALE_STEP_SIZE': 0.25,
         'TO_RGB': False},
 'BATCH_SIZE': 1,
 'DATALOADER': {'BUF_SIZE': 256, 'NUM_WORKERS': 8},
 'DATASET': {'DATA_DIM': 3,
             'DATA_DIR': './dataset/xiwa/',
             'IGNORE_INDEX': 255,
             'IMAGE_TYPE': 'rgb',
             'NUM_CLASSES': 5,
             'PADDING_VALUE': [127.5, 127.5, 127.5],
             'SEPARATOR': ' ',
             'TEST_FILE_LIST': './dataset/xiwa/test.txt',
             'TEST_TOTAL_IMAGES': 9,
             'TRAIN_FILE_LIST': './dataset/xiwa/train.txt',
             'TRAIN_TOTAL_IMAGES': 98,
             'VAL_FILE_LIST': './dataset/xiwa/val.txt',
             'VAL_TOTAL_IMAGES': 9,
             'VIS_FILE_LIST': './dataset/xiwa/test.txt'},
 'EVAL_CROP_SIZE': (512, 512),
 'FREEZE': {'MODEL_FILENAME': '__model__',
            'PARAMS_FILENAME': '__params__',
            'SAVE_DIR': 'freeze_model'},
 'MEAN': [0.5, 0.5, 0.5],
 'MODEL': {'BN_MOMENTUM': 0.99,
           'DEEPLAB': {'ASPP_WITH_SEP_CONV': True,
                       'BACKBONE': 'xception_65',
                       'BACKBONE_LR_MULT_LIST': None,
                       'DECODER': {'CONV_FILTERS': 256,
                                   'OUTPUT_IS_LOGITS': False,
                                   'USE_SUM_MERGE': False},
                       'DECODER_USE_SEP_CONV': True,
                       'DEPTH_MULTIPLIER': 1.0,
                       'ENABLE_DECODER': True,
                       'ENCODER': {'ADD_IMAGE_LEVEL_FEATURE': True,
                                   'ASPP_CONVS_FILTERS': 256,
                                   'ASPP_RATIOS': None,
                                   'ASPP_WITH_CONCAT_PROJECTION': True,
                                   'ASPP_WITH_SE': False,
                                   'POOLING_CROP_SIZE': None,
                                   'POOLING_STRIDE': [1, 1],
                                   'SE_USE_QSIGMOID': False},
                       'ENCODER_WITH_ASPP': True,
                       'OUTPUT_STRIDE': 16},
           'DEFAULT_EPSILON': 1e-05,
           'DEFAULT_GROUP_NUMBER': 32,
           'DEFAULT_NORM_TYPE': 'bn',
           'FP16': False,
           'HRNET': {'STAGE2': {'NUM_CHANNELS': [40, 80], 'NUM_MODULES': 1},
                     'STAGE3': {'NUM_CHANNELS': [40, 80, 160],
                                'NUM_MODULES': 4},
                     'STAGE4': {'NUM_CHANNELS': [40, 80, 160, 320],
                                'NUM_MODULES': 3}},
           'ICNET': {'DEPTH_MULTIPLIER': 0.5, 'LAYERS': 50},
           'MODEL_NAME': 'deeplabv3p',
           'MULTI_LOSS_WEIGHT': [1.0],
           'OCR': {'OCR_KEY_CHANNELS': 256, 'OCR_MID_CHANNELS': 512},
           'PSPNET': {'DEPTH_MULTIPLIER': 1, 'LAYERS': 50},
           'SCALE_LOSS': 'DYNAMIC',
           'UNET': {'UPSAMPLE_MODE': 'bilinear'}},
 'NUM_TRAINERS': 1,
 'SLIM': {'KNOWLEDGE_DISTILL': False,
          'KNOWLEDGE_DISTILL_IS_TEACHER': False,
          'KNOWLEDGE_DISTILL_TEACHER_MODEL_DIR': '',
          'NAS_ADDRESS': '',
          'NAS_IS_SERVER': True,
          'NAS_PORT': 23333,
          'NAS_SEARCH_STEPS': 100,
          'NAS_SPACE_NAME': '',
          'NAS_START_EVAL_EPOCH': 0,
          'PREPROCESS': False,
          'PRUNE_PARAMS': '',
          'PRUNE_RATIOS': []},
 'SOLVER': {'BEGIN_EPOCH': 1,
            'CROSS_ENTROPY_WEIGHT': None,
            'DECAY_EPOCH': [10, 20],
            'GAMMA': 0.1,
            'LOSS': ['lovasz_softmax_loss', 'softmax_loss'],
            'LOSS_WEIGHT': {'BCE_LOSS': 1,
                            'DICE_LOSS': 1,
                            'LOVASZ_HINGE_LOSS': 1,
                            'LOVASZ_SOFTMAX_LOSS': 0.2,
                            'SOFTMAX_LOSS': 0.8},
            'LR': 0.001,
            'LR_POLICY': 'poly',
            'LR_WARMUP': False,
            'LR_WARMUP_STEPS': 2000,
            'MOMENTUM': 0.9,
            'MOMENTUM2': 0.999,
            'NUM_EPOCHS': 50,
            'OPTIMIZER': 'adam',
            'POWER': 0.9,
            'WEIGHT_DECAY': 4e-05},
 'STD': [0.5, 0.5, 0.5],
 'TEST': {'TEST_MODEL': './saved_model/deeplabv3p_xception65_bn_xiwa/final'},
 'TRAIN': {'MODEL_SAVE_DIR': './saved_model/deeplabv3p_xception65_bn_xiwa/',
           'PRETRAINED_MODEL_DIR': './pretrained_model/deeplabv3p_xception65_bn_coco/',
           'RESUME_MODEL_DIR': '',
           'SNAPSHOT_EPOCH': 10,
           'SYNC_BATCH_NORM': False},
 'TRAINER_ID': 0,
 'TRAIN_CROP_SIZE': (512, 512)}
#Device count: 1
batch_size_per_dev: 1
W1120 15:26:22.734483  9616 device_context.cc:252] Please NOTE: device: 0, CUDA Capability: 75, Driver API Version: 10.2, Runtime API Version: 10.0
W1120 15:26:22.831225  9616 device_context.cc:260] device: 0, cuDNN Version: 7.6.
[SKIP] Shape of pretrained weight ./pretrained_model/deeplabv3p_xception65_bn_coco//logit/weights doesn't match.(Pretrained: (21, 256, 1, 1), Actual: (5, 256, 1, 1))
[SKIP] Shape of pretrained weight ./pretrained_model/deeplabv3p_xception65_bn_coco//logit/biases doesn't match.(Pretrained: (21,), Actual: (5,))
2020-11-20 15:26:27,787-WARNING: ./pretrained_model/deeplabv3p_xception65_bn_coco/.pdparams not found, try to load model file saved with [ save_params, save_persistables, save_vars ]
There are 730/732 varaibles in ./pretrained_model/deeplabv3p_xception65_bn_coco/ are loaded.
Use multi-thread reader
W1120 15:26:29.173449  9616 build_strategy.cc:170] fusion_group is not enabled for Windows/MacOS now, and only effective when running with CUDA GPU.

**Comment by nepeplwu** [comment]
@wangkan83 ，这种情况下显卡的利用率是不是一直为0？如果是的话，说明进程出了问题，建议关闭终端后再次启动训练进程

**Comment by jackie8310** [comment]
@nepeplwu ，显存的利用率50%，CUDA利用率80%，关闭终端后再次启动训练仍然出现上面情况！！

**Comment by nepeplwu** [comment]
@wangkan83 ，尝试使用`log_steps`指定一个较小的打印间隔看看
```cmd
python pdseg/train.py --cfg configs/deeplabv3p_xception65_xiwa.yaml --use_gpu --do_eval --log_steps 1 
```
